threescale_api/auth.py

```diff
@@ -4,33 +4,35 @@
 
 import requests
 
+
 class BaseClientAuth(requests.auth.AuthBase):
     "Abstract class for authentication of api client"
     def __init__(self, app, location):
         self.app = app
-        self.location = location
+        self.location = app.service.proxy.list().entity["credentials_location"]
```

Comment:
we need to consult this in person ;)

Reply:
I think this is resolved based on our conversation.